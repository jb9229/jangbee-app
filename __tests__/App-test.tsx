import 'react-native';

// import App from '../App';
import NavigationTestUtils from 'react-navigation/NavigationTestUtils';
import React from 'react';
import renderer from 'react-test-renderer';

describe('App snapshot', () => {
  // jest.useFakeTimers();
  // beforeEach(() => {
  //   NavigationTestUtils.resetInternalState();
  // });

  it('renders the loading screen', async () => {
    // const tree = renderer.create(< />).toJSON();
    // expect(tree).toMatchSnapshot();
  });

  // it('renders the root without loading screen', async () => {
  //   const tree = renderer.create(<App skipLoadingScreen />).toJSON();
  //   expect(tree).toMatchSnapshot();
  // });
});
