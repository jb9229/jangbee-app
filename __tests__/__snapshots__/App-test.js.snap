// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot renders the loading screen 1`] = `null`;

exports[`App snapshot renders the root without loading screen 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "overflow": "hidden",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        collapsable={false}
        pointerEvents="auto"
        removeClippedSubviews={false}
        style={
          Array [
            Object {
              "flex": 1,
              "overflow": "hidden",
            },
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
            Object {
              "opacity": 1,
            },
          ]
        }
      >
        <View
          style={
            Object {
              "flex": 1,
            }
          }
        >
          <View
            onLayout={[Function]}
            style={
              Object {
                "flex": 1,
              }
            }
          >
            <View
              enabled={false}
              hitSlop={
                Object {
                  "right": 50,
                }
              }
              maxDeltaY={20}
              minOffsetX={5}
              onGestureEvent={[Function]}
              onHandlerStateChange={[Function]}
            >
              <View
                style={
                  Object {
                    "backgroundColor": "#eee",
                    "flex": 1,
                    "flexDirection": "column-reverse",
                    "overflow": "hidden",
                  }
                }
              >
                <View
                  style={
                    Object {
                      "flex": 1,
                    }
                  }
                >
                  <View
                    pointerEvents="auto"
                    style={
                      Object {
                        "bottom": 0,
                        "left": 0,
                        "opacity": 1,
                        "position": "absolute",
                        "right": 0,
                        "top": 0,
                        "transform": Array [
                          Object {
                            "translateX": 0,
                          },
                          Object {
                            "translateY": 0,
                          },
                        ],
                      }
                    }
                  >
                    <View
                      accessibilityElementsHidden={false}
                      style={
                        Object {
                          "backgroundColor": "#fff",
                          "flex": 1,
                        }
                      }
                    >
                      <View
                        style={
                          Object {
                            "flex": 1,
                          }
                        }
                      >
                        <View
                          style={
                            Object {
                              "flex": 1,
                              "flexDirection": "row",
                            }
                          }
                        >
                          <Text>
                            업체명*
                          </Text>
                          <TextInput
                            allowFontScaling={true}
                            keyboardType="default"
                            onChangeText={[Function]}
                            placeholder="업체명을 입력해 주세요"
                            secureTextEntry={true}
                            style={Object {}}
                            underlineColorAndroid="transparent"
                            value=""
                          />
                        </View>
                        <View>
                          <Text
                            style={
                              Object {
                                "color": "red",
                              }
                            }
                          />
                        </View>
                        <View
                          style={
                            Object {
                              "flex": 1,
                              "flexDirection": "row",
                            }
                          }
                        >
                          <Text>
                            전화번호*
                          </Text>
                          <TextInput
                            allowFontScaling={true}
                            keyboardType="phone-pad"
                            onChangeText={[Function]}
                            placeholder="전화번호를 입력해 주세요"
                            secureTextEntry={true}
                            style={Object {}}
                            underlineColorAndroid="transparent"
                            value=""
                          />
                        </View>
                        <View>
                          <Text
                            style={
                              Object {
                                "color": "red",
                              }
                            }
                          />
                        </View>
                        <View
                          style={
                            Object {
                              "flex": 1,
                              "flexDirection": "row",
                            }
                          }
                        >
                          <Text>
                            비밀번호*
                          </Text>
                          <TextInput
                            allowFontScaling={true}
                            keyboardType="default"
                            onChangeText={[Function]}
                            placeholder="비밀번호를 입력해 주세요"
                            secureTextEntry={true}
                            style={Object {}}
                            underlineColorAndroid="transparent"
                            value=""
                          />
                        </View>
                        <View>
                          <Text
                            style={
                              Object {
                                "color": "red",
                              }
                            }
                          />
                        </View>
                        <View
                          style={
                            Object {
                              "flex": 1,
                              "flexDirection": "row",
                            }
                          }
                        >
                          <Text>
                            비밀번호 확인*
                          </Text>
                          <TextInput
                            allowFontScaling={true}
                            keyboardType="default"
                            onChangeText={[Function]}
                            placeholder="비밀번호를 재입력해 주세요"
                            secureTextEntry={true}
                            style={Object {}}
                            underlineColorAndroid="transparent"
                            value=""
                          />
                        </View>
                        <View>
                          <Text
                            style={
                              Object {
                                "color": "red",
                              }
                            }
                          />
                        </View>
                        <View>
                          <Text>
                            보유 장비*
                          </Text>
                          <TextInput
                            allowFontScaling={true}
                            onChangeText={[Function]}
                            onFocus={[Function]}
                            placeholder="보유 장비를 선택해 주세요"
                            underlineColorAndroid="transparent"
                            value=""
                          />
                        </View>
                        <View>
                          <Text
                            style={
                              Object {
                                "color": "red",
                              }
                            }
                          >
                            
                          </Text>
                        </View>
                        <View>
                          <Text>
                            업체주소(고객검색시 거리계산 기준이됨)
                          </Text>
                          <TextInput
                            allowFontScaling={true}
                            onChangeText={[Function]}
                            onFocus={[Function]}
                            placeholder="주소를 선택 해주세요"
                            underlineColorAndroid="transparent"
                            value=""
                          />
                        </View>
                        <View>
                          <Text
                            style={
                              Object {
                                "color": "red",
                              }
                            }
                          >
                            
                          </Text>
                        </View>
                        <View
                          style={
                            Object {
                              "flex": 1,
                              "flexDirection": "row",
                            }
                          }
                        >
                          <Text>
                            업체 상세주소
                          </Text>
                          <TextInput
                            allowFontScaling={true}
                            keyboardType="default"
                            onChangeText={[Function]}
                            placeholder="상세주소를 입력해 주세요"
                            secureTextEntry={true}
                            style={Object {}}
                            underlineColorAndroid="transparent"
                            value=""
                          />
                        </View>
                        <View>
                          <Text>
                            업체 소개
                          </Text>
                          <TextInput
                            allowFontScaling={true}
                            multiline={true}
                            numberOfLines={5}
                            onChangeText={[Function]}
                            placeholder="업체 소개를 해 주세요"
                            underlineColorAndroid="transparent"
                            value=""
                          />
                        </View>
                        <View>
                          <View
                            accessibilityRole="button"
                            accessibilityStates={Array []}
                            accessible={true}
                            isTVSelectable={true}
                            onResponderGrant={[Function]}
                            onResponderMove={[Function]}
                            onResponderRelease={[Function]}
                            onResponderTerminate={[Function]}
                            onResponderTerminationRequest={[Function]}
                            onStartShouldSetResponder={[Function]}
                            style={
                              Object {
                                "opacity": 1,
                              }
                            }
                          >
                            <View
                              style={
                                Array [
                                  Object {},
                                ]
                              }
                            >
                              <Text
                                style={
                                  Array [
                                    Object {
                                      "color": "#007AFF",
                                      "fontSize": 18,
                                      "padding": 8,
                                      "textAlign": "center",
                                    },
                                  ]
                                }
                              >
                                대표사진*
                              </Text>
                            </View>
                          </View>
                          <Text />
                        </View>
                        <View>
                          <Text
                            style={
                              Object {
                                "color": "red",
                              }
                            }
                          />
                        </View>
                        <View>
                          <View
                            accessibilityRole="button"
                            accessibilityStates={Array []}
                            accessible={true}
                            isTVSelectable={true}
                            onResponderGrant={[Function]}
                            onResponderMove={[Function]}
                            onResponderRelease={[Function]}
                            onResponderTerminate={[Function]}
                            onResponderTerminationRequest={[Function]}
                            onStartShouldSetResponder={[Function]}
                            style={
                              Object {
                                "opacity": 1,
                              }
                            }
                          >
                            <View
                              style={
                                Array [
                                  Object {},
                                ]
                              }
                            >
                              <Text
                                style={
                                  Array [
                                    Object {
                                      "color": "#007AFF",
                                      "fontSize": 18,
                                      "padding": 8,
                                      "textAlign": "center",
                                    },
                                  ]
                                }
                              >
                                작업사진1*
                              </Text>
                            </View>
                          </View>
                          <Text />
                        </View>
                        <View>
                          <Text
                            style={
                              Object {
                                "color": "red",
                              }
                            }
                          />
                        </View>
                        <View>
                          <View
                            accessibilityRole="button"
                            accessibilityStates={Array []}
                            accessible={true}
                            isTVSelectable={true}
                            onResponderGrant={[Function]}
                            onResponderMove={[Function]}
                            onResponderRelease={[Function]}
                            onResponderTerminate={[Function]}
                            onResponderTerminationRequest={[Function]}
                            onStartShouldSetResponder={[Function]}
                            style={
                              Object {
                                "opacity": 1,
                              }
                            }
                          >
                            <View
                              style={
                                Array [
                                  Object {},
                                ]
                              }
                            >
                              <Text
                                style={
                                  Array [
                                    Object {
                                      "color": "#007AFF",
                                      "fontSize": 18,
                                      "padding": 8,
                                      "textAlign": "center",
                                    },
                                  ]
                                }
                              >
                                작업사진2
                              </Text>
                            </View>
                          </View>
                          <Text />
                        </View>
                        <View>
                          <View
                            accessibilityRole="button"
                            accessibilityStates={Array []}
                            accessible={true}
                            isTVSelectable={true}
                            onResponderGrant={[Function]}
                            onResponderMove={[Function]}
                            onResponderRelease={[Function]}
                            onResponderTerminate={[Function]}
                            onResponderTerminationRequest={[Function]}
                            onStartShouldSetResponder={[Function]}
                            style={
                              Object {
                                "opacity": 1,
                              }
                            }
                          >
                            <View
                              style={
                                Array [
                                  Object {},
                                ]
                              }
                            >
                              <Text
                                style={
                                  Array [
                                    Object {
                                      "color": "#007AFF",
                                      "fontSize": 18,
                                      "padding": 8,
                                      "textAlign": "center",
                                    },
                                  ]
                                }
                              >
                                작업사진3
                              </Text>
                            </View>
                          </View>
                          <Text />
                        </View>
                        <View
                          style={
                            Object {
                              "flex": 1,
                              "flexDirection": "row",
                            }
                          }
                        >
                          <Text>
                            블로그
                          </Text>
                          <TextInput
                            allowFontScaling={true}
                            keyboardType="default"
                            onChangeText={[Function]}
                            placeholder="블로그 주소를 입력해 주세요"
                            secureTextEntry={true}
                            style={Object {}}
                            underlineColorAndroid="transparent"
                            value=""
                          />
                        </View>
                        <View
                          style={
                            Object {
                              "flex": 1,
                              "flexDirection": "row",
                            }
                          }
                        >
                          <Text>
                            SNG
                          </Text>
                          <TextInput
                            allowFontScaling={true}
                            keyboardType="default"
                            onChangeText={[Function]}
                            placeholder="SNS 주소를(또는 카카오톡 친구추가) 입력해 주세요"
                            secureTextEntry={true}
                            style={Object {}}
                            underlineColorAndroid="transparent"
                            value=""
                          />
                        </View>
                        <View
                          style={
                            Object {
                              "flex": 1,
                              "flexDirection": "row",
                            }
                          }
                        >
                          <Text>
                            홈페이지
                          </Text>
                          <TextInput
                            allowFontScaling={true}
                            keyboardType="default"
                            onChangeText={[Function]}
                            placeholder="홈페이지 주소를 입력해 주세요"
                            secureTextEntry={true}
                            style={Object {}}
                            underlineColorAndroid="transparent"
                            value=""
                          />
                        </View>
                        <View>
                          <View
                            accessibilityRole="button"
                            accessibilityStates={Array []}
                            accessible={true}
                            isTVSelectable={true}
                            onResponderGrant={[Function]}
                            onResponderMove={[Function]}
                            onResponderRelease={[Function]}
                            onResponderTerminate={[Function]}
                            onResponderTerminationRequest={[Function]}
                            onStartShouldSetResponder={[Function]}
                            style={
                              Object {
                                "opacity": 1,
                              }
                            }
                          >
                            <View
                              style={
                                Array [
                                  Object {},
                                ]
                              }
                            >
                              <Text
                                style={
                                  Array [
                                    Object {
                                      "color": "#007AFF",
                                      "fontSize": 18,
                                      "padding": 8,
                                      "textAlign": "center",
                                    },
                                  ]
                                }
                              >
                                저장
                              </Text>
                            </View>
                          </View>
                        </View>
                        <View
                          style={
                            Object {
                              "flex": 1,
                            }
                          }
                        >
                          <Modal
                            animationType="slide"
                            hardwareAccelerated={false}
                            onRequestClose={[Function]}
                            transparent={true}
                            visible={false}
                          >
                            <View
                              style={
                                Object {
                                  "backgroundColor": "#FFF",
                                  "flex": 1,
                                  "padding": 20,
                                }
                              }
                            >
                              <View
                                style={
                                  Object {
                                    "flex": 1,
                                    "flexDirection": "row",
                                  }
                                }
                              />
                              <View>
                                <View
                                  accessible={true}
                                  isTVSelectable={true}
                                  onResponderGrant={[Function]}
                                  onResponderMove={[Function]}
                                  onResponderRelease={[Function]}
                                  onResponderTerminate={[Function]}
                                  onResponderTerminationRequest={[Function]}
                                  onStartShouldSetResponder={[Function]}
                                  style={null}
                                >
                                  <Text
                                    style={null}
                                  >
                                    선택 완료
                                  </Text>
                                </View>
                              </View>
                            </View>
                          </Modal>
                        </View>
                        <View
                          style={
                            Object {
                              "flex": 1,
                            }
                          }
                        >
                          <Modal
                            animationType="slide"
                            hardwareAccelerated={false}
                            onRequestClose={[Function]}
                            transparent={true}
                            visible={false}
                          >
                            <View
                              style={
                                Object {
                                  "backgroundColor": "#FFF",
                                  "flex": 1,
                                  "padding": 20,
                                }
                              }
                            >
                              <View
                                style={
                                  Object {
                                    "flex": 1,
                                  }
                                }
                              >
                                <RCTWebView
                                  messagingEnabled={true}
                                  onLoadingError={[Function]}
                                  onLoadingFinish={[Function]}
                                  onLoadingStart={[Function]}
                                  onMessage={[Function]}
                                  onShouldStartLoadWithRequest={[Function]}
                                  scalesPageToFit={true}
                                  source={
                                    Object {
                                      "uri": "https://jb9229.github.io/postcode/add-map.html",
                                    }
                                  }
                                  style={
                                    Array [
                                      Object {
                                        "flex": 1,
                                      },
                                      Object {
                                        "backgroundColor": "#ffffff",
                                      },
                                      Object {
                                        "marginTop": 20,
                                      },
                                    ]
                                  }
                                />
                              </View>
                            </View>
                          </Modal>
                        </View>
                      </View>
                    </View>
                  </View>
                </View>
                <View
                  onLayout={[Function]}
                  pointerEvents="box-none"
                  style={
                    Object {
                      "left": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "backgroundColor": "#FFF",
                        "transform": Array [
                          Object {
                            "translateX": -750,
                          },
                        ],
                      }
                    }
                  >
                    <View
                      onLayout={[Function]}
                      pointerEvents="box-none"
                      style={
                        Object {
                          "backgroundColor": "#FFF",
                          "borderBottomColor": "#A7A7AA",
                          "borderBottomWidth": 0.5,
                          "height": 64,
                          "paddingBottom": 0,
                          "paddingLeft": 0,
                          "paddingRight": 0,
                          "paddingTop": 20,
                        }
                      }
                    >
                      <View
                        style={
                          Object {
                            "flex": 1,
                          }
                        }
                      />
                    </View>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
    <View
      onLayout={[Function]}
      pointerEvents="box-none"
      style={
        Object {
          "backgroundColor": "#fff",
          "borderTopColor": "rgba(0, 0, 0, .3)",
          "borderTopWidth": 0.5,
          "flexDirection": "row",
          "height": 49,
          "paddingBottom": 0,
          "paddingLeft": 0,
          "paddingRight": 0,
          "paddingTop": 0,
        }
      }
    >
      <View
        accessibilityLabel="Home, tab, 1 of 3"
        accessible={true}
        hitSlop={
          Object {
            "bottom": 5,
            "left": 15,
            "right": 15,
            "top": 5,
          }
        }
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
            },
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "flexDirection": "column",
              "justifyContent": "flex-end",
            },
            undefined,
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "height": 29,
              },
              false,
              Object {
                "flex": 1,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "height": "100%",
                "justifyContent": "center",
                "minWidth": 25,
                "opacity": 1,
                "position": "absolute",
                "width": "100%",
              }
            }
          >
            <Text />
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "height": "100%",
                "justifyContent": "center",
                "minWidth": 25,
                "opacity": 0,
                "position": "absolute",
                "width": "100%",
              }
            }
          >
            <Text />
          </View>
        </View>
        <Text
          allowFontScaling={true}
          numberOfLines={1}
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#007AFF",
              "fontSize": 11,
              "marginBottom": 1.5,
              "textAlign": "center",
            }
          }
        >
          Home
        </Text>
      </View>
      <View
        accessibilityLabel="Links, tab, 2 of 3"
        accessible={true}
        hitSlop={
          Object {
            "bottom": 5,
            "left": 15,
            "right": 15,
            "top": 5,
          }
        }
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
            },
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "flexDirection": "column",
              "justifyContent": "flex-end",
            },
            undefined,
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "height": 29,
              },
              false,
              Object {
                "flex": 1,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "height": "100%",
                "justifyContent": "center",
                "minWidth": 25,
                "opacity": 0,
                "position": "absolute",
                "width": "100%",
              }
            }
          >
            <Text />
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "height": "100%",
                "justifyContent": "center",
                "minWidth": 25,
                "opacity": 1,
                "position": "absolute",
                "width": "100%",
              }
            }
          >
            <Text />
          </View>
        </View>
        <Text
          allowFontScaling={true}
          numberOfLines={1}
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#8E8E93",
              "fontSize": 11,
              "marginBottom": 1.5,
              "textAlign": "center",
            }
          }
        >
          Links
        </Text>
      </View>
      <View
        accessibilityLabel="Settings, tab, 3 of 3"
        accessible={true}
        hitSlop={
          Object {
            "bottom": 5,
            "left": 15,
            "right": 15,
            "top": 5,
          }
        }
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
            },
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "flexDirection": "column",
              "justifyContent": "flex-end",
            },
            undefined,
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "height": 29,
              },
              false,
              Object {
                "flex": 1,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "height": "100%",
                "justifyContent": "center",
                "minWidth": 25,
                "opacity": 0,
                "position": "absolute",
                "width": "100%",
              }
            }
          >
            <Text />
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "height": "100%",
                "justifyContent": "center",
                "minWidth": 25,
                "opacity": 1,
                "position": "absolute",
                "width": "100%",
              }
            }
          >
            <Text />
          </View>
        </View>
        <Text
          allowFontScaling={true}
          numberOfLines={1}
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#8E8E93",
              "fontSize": 11,
              "marginBottom": 1.5,
              "textAlign": "center",
            }
          }
        >
          Settings
        </Text>
      </View>
    </View>
  </View>
</View>
`;
